test/screen_checker/fetch_test.exs

```diff
@@ -1,8 +1,8 @@
-defmodule ScreenChecker.FetchTest do
+defmodule ScreenChecker.SolariData.FetchTest do
   use ExUnit.Case
 
   import Mock
-  alias ScreenChecker.{Fetch, Ping}
+  alias ScreenChecker.SolariData.{Fetch, Ping}
```

Comment:
were there assocaited GDS tests that could be moved over as well?

Reply:
No 😬